scheds/rust/scx_chaos/src/bpf/main.bpf.c

```diff
@@ -457,15 +463,38 @@ void BPF_STRUCT_OPS(chaos_runnable, struct task_struct *p, u64 enq_flags)
 	return p2dq_runnable_impl(p, enq_flags);
 }
 
+void BPF_STRUCT_OPS(chaos_running, struct task_struct *p)
+{
+	struct chaos_task_ctx *taskc;
+
+	if (!(taskc = lookup_create_chaos_task_ctx(p))) {
+		scx_bpf_error("failed to lookup task context in enqueue");
+		return;
+	}
+
+	p2dq_running_impl(p);
+
+	s32 task_cpu = scx_bpf_task_cpu(p);
+	if (taskc->next_trait == CHAOS_TRAIT_CPU_FREQ) {
+		if (cpu_freq_min > 0) {
```

Comment:
is 0 not a valid option here meaning "minimum frequency"?

Reply:
IIRC it's a noop